guardian/manage-frontend
# Create new Supporter plus Update Amount component

## What does this change?

Create a new component for updating the amount a supporter pays for Supporter plus. Updates the supporter plus configuration to include a maximum price per currency.



## How to test

Enable the feature flag (`withFeature=supporterPlusUpdateAmount`) and click `Manage recurring support` on the Monthly + Extras product card. Click `Change amount` and see the new component.

(Confirming the amount change will not work as of this PR since the server side work is still todo)



## How can we measure success?



## Have we considered potential risks?



## Images

<img width="890" alt="image" src="https://github.com/guardian/manage-frontend/assets/114918544/d5d39cd7-ef55-41de-9629-c76e87b064b7"><img width="903" alt="image" src="https://github.com/guardian/manage-frontend/assets/114918544/82190504-702e-4268-978e-f75e66b14e19">
Validation error

<img width="542" alt="image" src="https://github.com/guardian/manage-frontend/assets/114918544/8c750eaf-38b6-42c2-a3b3-d7c90203a684">


## Accessibility
